tools/please_pex/BUILD

```diff
@@ -1,7 +1,69 @@
-# TODO(jpoole): This should be a remove file on the plugin repo's release page and we should remove //_please:please_pex
-filegroup(
+subinclude("//build_defs:python")
```

Comment:
Would suggest removing the top level `tools` dir, I'm not sure it adds much now it's in its own repo